### PR TITLE
Update Windows Installation and other improvements

### PR DESCRIPTION
**Pull request by Sae86**
Signed-off-by: Sae86 <sara.batllori@intel.com>

### CODE REVIEW

File: docs/sphinx/Download.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+--------
+
+You can find the latest release here:
+
+.. image:: _images/LR.png
+   :target: https://github.com/chipsec/chipsec/releases/latest
+
+Older releases can be found `here <https://github.com/chipsec/chipsec/releases>`_
+
+After downloading there are some steps to follow to build the driver and run, please refer to Installation and Using CHIPSEC sections.
```
Comment (npmitche) [inline docs/sphinx/Download.rst:22]: Can we add a link to the Installation section?

File: docs/sphinx/Install in Windows.rst [hunk 2]
```diff
@@ -59,26 +59,26 @@ In CMD shell:
...
    - Navigate: Troubleshooting > Advanced Settings > Startup Settings > Reboot 
-   - After reset choose F7 “Disable driver signature checks”
+   - After reset choose F7 or 7 “Disable driver signature checks”
 
 Method 2: 
 
    - Disable Secure Boot in the BIOS setup screen then disable driver signature checks as in Windows 8 with Secure Boot disabled
 
-**Windows 7 64-bit / Windows Server 2008 64-bit / Windows 8 (with Secure Boot disabled) / Windows Server 2012 (with Secure Boot disabled)):**
+**Windows 8 (with Secure Boot disabled) / Windows Server 2012 (with Secure Boot disabled)):**
```
Comment (npmitche) [inline docs/sphinx/Install in Windows.rst:72]: This also works on Win10.

**Review by npmitche — APPROVED** [review]

File: docs/sphinx/Contact.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+For any questions or suggestions please contact us at: chipsec@intel.com
+
+We also have the `issue tracker <https://github.com/chipsec/chipsec/issues>`_ in our GitHub repo. If you'd like to report a bug or make a request please open an issue.
+
+If you'd like to make a contribution to the code please open a `pull request <https://github.com/chipsec/chipsec/pulls>`_
+
+Mailing lists:
+
+ * CHIPSEC users: https://groups.google.com/forum/#!forum/chipsec-users
```
Comment (mjkyung) [inline docs/sphinx/Contact.rst:12]: Just checking - I can't access this address. Is it for subscription only?

**Review by frinzell — APPROVED** [review]

File: docs/sphinx/Contact.rst (hunk 3 above)
Comment (npmitche) [reply docs/sphinx/Contact.rst:12]: The group is there, but the permissions got messed up and we need @johnloucaides to help us fix it as he's the owner.

**Pull request merged**